# DAVID Gene Ontology Enrichment Analysis results

The [gene expression data](
http://www.nature.com/nature/journal/v518/n7539/full/nature14252.html#supplementary-information) 
from the 2015 Gjoneska Nature paper, 
[***Conserved epigenomic signals in mice and humans reveal immune basis of Alzheimer's disease***](
http://www.nature.com/nature/journal/v518/n7539/full/nature14252.html).
([Google scholar](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C33&q=Conserved+epigenomic+signals+in+mice+and+humans+reveal+immune+basis+of+Alzheimer%E2%80%99s+disease&btnG=))
is used to perform Gene Ontology Enrichment Analyses (GOEAs)

# DAVID versions

  * DAVID versions used:    
    * DAVID v6.7 (Jan 2010) and DAVID Knowledgebase (Sep 2009)    
    * DAVID v6.8 (Oct 2016) and DAVID Knowledgebase (May 2016)    
  * DAVID version information:    
    * [DAVID Versions](https://david.ncifcrf.gov/content.jsp?file=release.html)    
    * [DAVID Knowledgebase versions](https://david.ncifcrf.gov/content.jsp?file=update.html)    

# Gene lists
The genes used in the DAVID GOEA runs were obtained from the 
supplemental table, nature14252-s1.xlsx:

  * genes_Consistent_Decrease.txt
  * genes_Consistent_Increase.txt
  * genes_Late_Decrease.txt
  * genes_Late_Increase.txt
  * genes_population.txt
  * genes_Transient_Decrease.txt
  * genes_Transient_Increase.txt

Copyright (C) 2016-2018, DV Klopfenstein, Haibao Tang, et al. All rights reserved.
